### ISSUE TITLE
Problem with mouse cursor after resizing part of the UI

### ISSUE THREAD

**Issue by luongvo209** [BoostIO/Boostnote#556]
My OS: Ubuntu 16.04.2  
My boostnote version: latest 0.8.9  

Problem: After resizing (or specifically, enlarging) part of the UI, the mouse cursor behaves abnormally. I cannot reach some area of my text. More in below video: 
https://gfycat.com/ScratchyBountifulGecko

**Comment by kazup01** [comment]
I added to [roadmap](https://github.com/BoostIO/Boostnote/projects/3)!

**Comment by alexrussell** [comment]
FWIW I see this too. To expand on what @LewisVo says, it appears to happen when you cause the size of the editor to change by resizing the other UI elements (folders/search) or using the fullscreen/distraction-free button (that removes the sidebars) *and* if that causes the content to change its height due to word wrapping.

As a user, you can "fix" this immediately by resizing the window itself. This appears to make the text editor repaint itself and work out where the cursor should be is.

I imagine a programmatical fix for this would lie somewhere in that - somehow invalidate the state of the CodeMirror editor when the internal UI changes size (even if the window doesn't) the same as you presumably do when the window itself is resized.

**Comment by zzdjk6** [comment]
I have the same issue

**Comment by alexrussell** [comment]
This appears to be fixed for me now. I've just updated from 0.11.9 where the bug was happening to 0.11.10 where it is not (and now I'm at 0.11.11 where it is still fixed for me).

I can't see any mention of it being officially fixed in the changelog, and obviously this issue has gone untouched, so I'm guessing it was fixed by accident.

Can @luongvo209  and @zzdjk6 confirm that it's now fixed, and then we can close this off.
